README.md

```diff
@@ -26,21 +26,27 @@ Many Windows developers have found that setting up an Ubuntu virtual machine is
 1. Set up [RBENV](https://github.com/sstephenson/rbenv#homebrew-on-mac-os-x) and Ruby 2.0
   1. Add `if which rbenv > /dev/null; then eval "$(rbenv init -)"; fi` to `~/.profile`
   1. source `~/.profile`
-  1. rbenv install 2.0.0-p451
-  1. rbenv global 2.0.0-p451
-  1. rbenv rehash
+  1. `rbenv install 2.0.0-p451`
+  1. `rbenv global 2.0.0-p451`
+  1. `rbenv rehash`
 
 ### Ubuntu 14.04
 
+1. `curl -sL https://deb.nodesource.com/setup | sudo bash -`
```

Comment:
Can we put a description of what this step does as well?  This is probably needed in lots more places in these instructions, but I feel like we owe an explanation when we ask people to pull and execute a script from a location we don't control.